### DISCUSSION TITLE
MariaDB Support

### DISCUSSION THREAD

**Discussion by bkincz** [payloadcms/payload#9390] [Feature Requests & Ideas]
Hey,

We've been looking to replace Wordpress and Payload was one of the best options we found but there is only one hurdle that I'd need to pass to be able to make the switch, which is MariaDB database engine support.

It would be great if we could get MariaDB support 🙌

**Comment by DanRibbens** [comment]
We will need drizzle to support MariaDB since that's our ORM outside of mongodb. Looks like it's already on their roadmap: https://github.com/drizzle-team/drizzle-orm/discussions/2436&ved=2ahUKEwiqxoHDh-yJAxVHQjABHXHLDToQjjh6BAg6EAE&usg=AOvVaw1YcwiZFJmDqvXghNk1kJjp

**Reply by bkincz** [reply]
Ah fair enough, guess this will be a waiting game then
Thanks!
